src/game_battler.h

```diff
@@ -898,4 +906,8 @@ inline void Game_Battler::SetDirectionFlipped(bool flip) {
 	direction_flipped = flip;
 }
 
+inline int Game_Battler::GetBaseAttributeRate(int attribute_id) const {
+	return 2;
```

Comment:
> Refactor Battler attributes 

Does this have a use? It is overoaded for both actor and enemy.
Also needs a ``// C - default`` comment

Reply:
Removed this function